hack/build.sh

```diff
@@ -92,11 +94,13 @@ source_format() {
      echo "✋   go get golang.org/x/tools/cmd/goimports"
      echo "✋ to enable import cleanup. Import cleanup skipped."
 
-     # Run go fmt insteat
+     # Run go fmt instead
      go_fmt
   else
      echo "🧽 ${S}Format"
-     goimports -w cmd pkg
+     goimports -w $(find -name '*.go' | grep -v vendor)
+     echo "🧽 ${S}Format: Simplify code.."
+     gofmt -s -w $(find -path './vendor' -prune -o -type f -name '*.go' -print)
```

Comment:
This find as well as above will break on filenames which contain spaces.

Also, you need to be careful as e.g. `-prune` is not available for macOs' find.

```
find -path './vendor' -prune -o -type f -name '*.go' -print
find: illegal option -- p
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
```

Reply:
good catch!

Wonder if the clients should have provision in CI system to test against different platforms we ship the binaries for..